src/List.h

```diff
@@ -123,7 +123,7 @@ class List {
 		}
 
 	// Return nth ent of list (do not remove).
-	T& operator[](int i) const
+	T& operator[](unsigned int i) const
```

Comment:
I'd be fine with changing this one to a `size_t`. That's how the standard implements it.